pkg/inctimer/inctimer_test.go

```diff
@@ -21,6 +21,18 @@ import (
 	"time"
 )
 
+func TestTimerAfter(t *testing.T) {
+	for i := 0; i < 100000; i++ {
```

Comment:
nit: makes it easier to read
```suggestion
	for i := 0; i < 100_000; i++ {
```